src/NodaTime/CalendarSystem.cs

```diff
@@ -189,6 +191,7 @@ public static CalendarSystem ForId(string id)
         /// this implementation align exactly with the BCL implementation.
         /// </remarks>
         /// <returns>A Persian calendar system.</returns>
+        [Obsolete("Use the PersianSimple property for compatibility with 2.0.")]
```

Comment:
Mmm. It's a tricky one. I think the word "obsolete" is unfortunate in general for this. Will leave it as is right now, but let's revisit this when we have some more details.